integration_test/oracle_module/verify_penalty_counts.yaml

```diff
@@ -0,0 +1,46 @@
+- name: Test penalty counts for oracle process
+  inputs:
+    # Get slashing window
+    - cmd: seid q oracle params --output json | jq -r ".params.slash_window"
+      env: ORACLE_SLASH_WINDOW
+    # Get vote period
+    - cmd: seid q oracle params --output json | jq -r ".params.vote_period"
+      env: VOTE_PERIOD
+    # Get curent height
+    - cmd: seid q block | jq -r ".block.header.height"
+      env: BLOCK_HEIGHT
+    # calculate progress in slash window
+    - cmd: expr $BLOCK_HEIGHT % $ORACLE_SLASH_WINDOW
+      env: WINDOW_PROGRESS
+    # get expected number of penalty counts
+    - cmd: expr $WINDOW_PROGRESS / $VOTE_PERIOD
+      env: EXPECTED_COUNTS
+    # Get admin val addr
+    - cmd: printf "12345678\n" | seid keys show node_admin --bech val --output json | jq -r ".address"
+      env: ADMIN_VAL_ADDR
+    # get vote penalty counts
+    - cmd: seid q oracle vote-penalty-counter $ADMIN_VAL_ADDR --output json | jq -r ".vote_penalty_counter"
+      env: PENALTY_JSON
+    # get success counts
+    - cmd: echo $PENALTY_JSON | jq -r ".success_count"
+      env: SUCCESS_COUNT
+    # get abstain counts
+    - cmd: echo $PENALTY_JSON | jq -r ".abstain_count"
+      env: ABSTAIN_COUNT
+    # get miss counts
+    - cmd: echo $PENALTY_JSON | jq -r ".miss_count"
+      env: MISS_COUNT
+    # calculate total
+    - cmd: expr $SUCCESS_COUNT + $ABSTAIN_COUNT + $MISS_COUNT
+      env: TOTAL_COUNT
+    # calculate success rate
+    - cmd: expr $(expr $SUCCESS_COUNT \* 100 ) / $TOTAL_COUNT
+      env: SUCCESS_PERCENT
+  verifiers:
+    # verify total counts GEQ expected: not asserting equal due to block movement between evaluation
```

Comment:
Can we add a sleep and then make sure TOTAL_COUNT > EXPECTED_COUNTS?

Reply:
why do we need to add a sleep? the expected counts is determined by the current block heights (and we'd want the total count to be exactly equal IF no block progress between the queries, but since they could progress we check for GEQ in cas a block or two have passed)